lib/config/migrations/migrations-service.spec.ts

```diff
@@ -0,0 +1,45 @@
+import type { RenovateConfig } from '../types';
+import { MigrationsService } from './migrations-service';
+
+describe('config/migrations/migrations-service', () => {
+  it('should remove deprecated properties', () => {
+    for (const property of MigrationsService.removedProperties) {
+      const originalConfig: RenovateConfig = {
+        [property]: 'test',
+      };
+
+      const migratedConfig = MigrationsService.run(originalConfig);
+      expect(migratedConfig[property]).toBeUndefined();
```

Comment:
I think we should use `not.toHaveProperty` matcher?

Your code will not catch properties with `undefined` value assignment.